### DISCUSSION TITLE
In-game PP Leaderboards

### DISCUSSION THREAD

**Discussion by aireulain** [ppy/osu#13024] [Ideas]
**Describe the new feature:** PP leaderboard in-game which sorts scores on the map by highest pp achieved

**Proposal designs of the feature:** [Something like this](https://i.imgur.com/lkipe3O.png)

**Comment by nbvdkamp** [comment]
Might be better to have this as a separate option from the global/country/etc. leaderboards so that it can be applied to non global leaderboards too, would also make it easier to then add more kinds of leaderboards like acc or combo.

**Comment by EVAST9919** [comment]
In most cases Higher score = Higher PP.
Imo it's not worth it

**Reply by Givikap120** [reply]
pp is more accurate than score in 99% of the cases
so it's score that should be removed if anything

**Comment by awesomecuber** [comment]
IMO the end goal should be to merge pp and score into one system. Seems kinda dumb to me that it's possible to get a higher score but less pp.

**Comment by peppy** [comment]
I’d hope this is no longer required with lazer’s scoring method.

**Reply by Givikap120** [reply]
pp is solving >90% of the balancing problems with multipliers
including those that can't be fixed in multipliers at all (such as EZ making map harder but decreasing score in 2x regardless)
so this is always required as pp will always be more accurate than score and it's not even close

**Comment by huoyaoyuan** [comment]
> In most cases Higher score = Higher PP.

My personal thoughts:
- Scoring should be very stable, while PP may change when adjusting the algorithm.
- When applying the same mods, good algorithms should ensure higher score = higher PP. But when **comparing across mods**, this will probably not be true.

**Comment by hostilew** [comment]
This was brought up in the infrastructure plan so I'm bumping this thread for discussion, hopefully that's okay. This seems like the only way to balance leaderboards with so many mod options like rate changes and individual difficulty adjustments. There should be a priority for continuing to reward harder plays with higher leaderboard placements, and the gamut of possible modded plays for any given map is too wide to split into a reasonable number of separate leaderboards on lazer. (Each mod combo getting it's own leaderboard on stable is already a lot.) Leaderboard sorting that actually analyzes the map/play and determines how good the play is, is preferable to any blind system for competitive play.

**Comment by peppy** [comment]
not the answer you're looking for.

**Comment by TextAdventurer12** [comment]
this is probably going to be even more desired with the new combo scaling removal rework, since score and pp now have very little correlation

**Reply by Givikap120** [reply]
Actually lazer score and pp now have more correlation (at least for the same mod combination)

**Comment by Siiiuuuuu** [comment]
How is this still not implemented... 😭  This pretty much solves most if not all problems regarding score multiplier issues and what not. Especially with rates and DA on the horizon, this is a must have feature.

**Reply by Siiiuuuuu** [reply]
Obviously if implemented should show the pp number on the leaderboard for each score, and maybe even add the option to replace the score for pp for addiotional declutter.

**Comment by MizuIro-H** [comment]
Well, it will be more useful as the Combe scaling is removed from standard pp-algorithm, as more score ≠ more pp.
